core/components/molecules/fileUploader/__stories__/index.story.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import { text, boolean } from '@storybook/addon-knobs';
+import { FileUploader, FileUploaderList, Link } from '@/index';
+import { FileUploaderComponent, FileUploaderListComponent } from './_common_/types';
+
+export const all = () => {
+  const [fileNames, setFileNames] = React.useState<any>([]);
+
+  const title = text('title', 'Upload files');
+  const formatLabel = text('Format Label', 'Accepted formats: PDF, jpg');
+  const sizeLabel = text('Max size label', 'Maximum size: 25 MB');
```

Comment:
Please use props name instead of free text as knob label, i.e. `Format Label` should be `formatLabel`.